### ISSUE TITLE
missing alt texts

### ISSUE THREAD

**Issue by OniriCorpe** [YunoHost/pepettes#8]
alt text is missing for the logo and for the currency and frequency dropdowns...

**Comment by OniriCorpe** [comment]
done with 89dee6880703b10afc60f53a16e0769511f23080 & 62ba1edb0fae8ab3544b5b4686fa1d9250da397a
